lib/cmds/config_cmds/add.js

```diff
@@ -42,7 +42,16 @@ module.exports.builder = yargs => {
         'Pass proxy config as raw config instead of creating a httpsAgent',
       type: 'boolean'
     })
+    .requiresArg([
+      'management-token',
+      'active-space-id',
+      'active-environment-id'
+    ])
     .config('config', 'set all the config options at once')
+    .fail(msg => {
```

Comment:
We should consider using the logging utility from `lib/utils/log.js` -> `logError` instead of calling `console.log` directly.